newsAppView.js

```diff
@@ -0,0 +1,78 @@
+class NewsAppView {
+
+  constructor(model, api) {
+    this.model = model;
+    this.api = api;
+    this.mainContainerEl = document.querySelector("#main-container");
+  }
+
+  loadArticles() {
+    this.api.getArticles((articles) => {
+      model.setArticles(articles)
+    })
+  }
+
+  displayArticles() {
+    let articles = this.model.articles;
+    let id = 0;
+    for (const article of articles) {
+      let newArticle = document.createElement("div");
+      newArticle.className = "article";
+      newArticle.setAttribute("id", id);
+      this.createHeadline(article, newArticle, id);
```

Comment:
Instead of using methods to create the headline you could use innerHTML here and template literals to grab the parts of the fetch data you need to reduce the amount of code?